changelog/README.md

```diff
@@ -12,6 +16,10 @@
 * [Get all availability adjustments](../operations/availabilityadjustments.md#get-all-availability-adjustments)
   * Extended response object with `FirstTimeUnitReleaseUtc`
 
+## 25th February 2025
+* [Get all cashier transactions)](../operations/cashiertransactions.md#get-all-cashier-transactions):
+  * Extended request object with `ActualEndUtc` filtering parameter.

```

Comment:
issue: Per guidelines, the changelog date should reflect the date when the documentation changed, not when the feature was introduced, so please move it under the latest update.